netbox/templates/login.html

```diff
@@ -46,7 +46,21 @@ <h2 class="text-center mb-4">{% trans "Log In" %}</h2>
               <input type="hidden" name="next" value="{{ request.POST.next }}" />
             {% endif %}
 
-            {% render_form form %}
+            <div class="form-group">
+            <label for="id_username" style="font-weight: bold; margin-bottom: 5px;">{{ form.username.label }}</label>
+            {{ form.username }}
+            {% for error in form.username.errors %}
+            <div class="alert alert-danger">{{ error }}</div>
+            {% endfor %}
+            </div>
+
+            <div class="form-group">
```

Comment:
There isn't enough space between the two form fields. Try adding the `mb-3` class or similar to the first group.

![screenshot](https://github.com/netbox-community/netbox/assets/13487278/9cde5a24-f163-4d96-95f8-80e50b4e6ec5)